include/zephyr/sys/util_internal.h

```diff
@@ -49,6 +49,8 @@
  */
 #define Z_IS_ENABLED3(ignore_this, val, ...) val
 
+#define Z_IS_EQ(_0, _1) Z_HAS_COMMA(Z_CAT4(Z_IS_, _0, _EQ_, _1)())
```

Comment:
Seems like a comment will be helpful. Something like.

/* Implementation of IS_EQ(). Returns 1 if _0 and _1 are the same integer from 0 to 255, 0 otherwise. */

Reply:
Thanks for the suggestion, fixed.